XamlAnimatedGif.Shared/Properties/VersionInfo.cs

```diff
@@ -9,7 +9,7 @@ namespace XamlAnimatedGif.Properties
 {
     class VersionInfo
     {
-        public const string Version = "1.0.9.0";
+        public const string Version = "1.0.10.0";
```

Comment:
Adding support for Silverlight deserves more than just a new revision number ;)
Should probably be 1.1.0.0